Tests/Common/Data/TradeBarConsolidatorTests.cs

```diff
@@ -130,11 +130,11 @@ public void OneMinuteAlwaysFiresEveryTimeOnMinueDataExceptFirstPoint()
 
             consolidator.Update(new TradeBar {Close = 2m, Time = reference.AddMinutes(1)});
             Assert.IsNotNull(consolidated);
-            Assert.AreEqual(2, consolidated.Close);
+            Assert.AreEqual(1, consolidated.Close);
 
             consolidator.Update(new TradeBar {Close = 3m, Time = reference.AddMinutes(2)});
             Assert.IsNotNull(consolidated);
-            Assert.AreEqual(3, consolidated.Close);
+            Assert.AreEqual(2, consolidated.Close);
```

Comment:
OK. See commit d4cc904. 

So, the Pull Request should include 3 commits:  0a2cab1, 60fb3ac and d4cc904.